Sources/PackageLoading/CMakeLists.txt

```diff
@@ -17,12 +17,15 @@ add_library(PackageLoading
   Target+PkgConfig.swift
   TargetSourcesBuilder.swift
   ToolsVersionLoader.swift
-  UserManifestResources.swift)
+  UserManifestResources.swift
+  ../Workspace/WindowsToolchainInfo.swift)
```

Comment:
It seems like the wrong abstraction to re-include this file here, perhaps it should be sunk down to a lower-level target?

Reply:
Oh, if there is a lower level library that I can re-use, it would be great.  What library do you suggest?  I really don't want to create a new library for just that.

Reply:
Ah, actually I think you can do what you want via the protocol I mentioned in the other comment. Basically, compute the flags in the UserToolchain (as you're doing already) and then plumb them to manifest loader via `UserManifestProvider` struct. That is how manifest loader is currently getting swift compiler and some other paths. See this for reference: https://github.com/apple/swift-package-manager/blob/master/Sources/Workspace/UserToolchain.swift#L293-L298

Reply:
I don't understand how that would work.  The implementation is in `PackageLoading` which `Workspace` doesn't link against.

Reply:
...chatted with Saleem on Slack to walkthrough what I meant in more detail...